### DISCUSSION TITLE
IAO TestSuite v1.1.3-release

### DISCUSSION THREAD

**Discussion by IshraqueSarwar** [IshraqueSarwar/IAO-Test-suite-WinBuild#1] [Announcements]
Introducing IAO TestSuite! 
This is the first production version of the program.
You can read more about it and have a peek at the subjects that are available [here](https://github.com/IshraqueSarwar/IAO-Test-suite-WinBuild#readme)
If you have any questions, comments, or concerns, you can post them in the discussion.

<hr /><em>This discussion was created from the release <a href='https://github.com/IshraqueSarwar/IAO-Test-suite-WinBuild/releases/tag/v1.1.3-release'>IAO TestSuite v1.1.3-release</a>.</em>

**Comment by nasseef20** [comment]
Looking forward to osx support!
